en/faq/accounts/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Accounts
+---
+
+# FAQ: Accounts and settings
+
+[[toc]]
+
+## How do accounts work on Dodona?
+
+Accounts on Dodona are automatically created when you log in for the first time. You can login with your institutional account (e.g. UGent, Smartschool, ...) or with a personal account (e.g. Google, Microsoft, ...). If you use an institutional account, your account is automatically linked to your school or university.
+
+It is not possible to manually create an account on Dodona with a username and password.
+
+## I work in several schools, how does this work?
+
+Your account on Dodona is linked to the way you log in. For example, if you have Smartschool accounts at several schools, each of them will be linked to a different Dodona account.
+
+## I get an error about linked accounts when I try to sign in, what should I do?
+
+Your Dodona account is linked to the way you login. However, some schools use more than one way to log in. For example, they have both Smartschool and Microsoft 365. To prevent students from these schools from ending up with multiple accounts on Dodona, we can (manually) link multiple sign-in methods to a single school.
```

Comment:
```suggestion
Your Dodona account is linked to the way you log in. However, some schools use more than one way to log in. For example, they have both Smartschool and Microsoft 365. To prevent students from these schools from ending up with multiple accounts on Dodona, we can (manually) link multiple sign-in methods to a single school.
```